src/odemis/driver/actuator.py

```diff
@@ -666,6 +667,27 @@ def _get_rot_matrix(self, invert=False):
             rotation *= -1
         return RigidTransform(rotation=rotation).matrix
 
+    def _getAxesRange(self):
+        """
+        Calculate the axes range of position VA from the given range of dep's.
+        """
+        # Calculate rectangular border points on the given range
+        min_x, max_x = self.axes["x"].range
+        min_y, max_y = self.axes["y"].range
+        coordinates = [[min_x, min_y],
+                       [min_x, max_y],
+                       [max_x, min_y],
+                       [max_x, max_y]]
+
+        # Calculate range based on the conversion
+        new_coordinates = list(map(self._convertPosFromdep, coordinates))
+
+        # Update default axes range
+        new_range = numpy.array(new_coordinates, dtype=float)
+        self.axes["x"].range = (min(new_range[:, 0]), max(new_range[:, 0]))
+        self.axes["y"].range = (min(new_range[:, 1]), max(new_range[:, 1]))
+
+
```

Comment:
Normally there is only 1 line between each method => delete one empty line.

Reply:
Sorry for this mistake